### PR TITLE
new package: pueue-0.6.1

### PR DESCRIPTION
**Pull request by crater2150**

### CODE REVIEW

File: srcpkgs/pueue/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+build_style=cargo
+short_desc="Command-line tool for managing long-running tasks"
+maintainer="crater2150 <void@qwertyuiop.de>"
+license="MIT"
+homepage="https://github.com/Nukesor/pueue"
+distfiles="https://github.com/Nukesor/pueue/archive/v${version}.tar.gz"
+checksum=84562b76cbf7d3bec7dce7337d15901ef5a5a750e749ca1aecb7bb6ff0385ce7
+
+post_install() {
+        vlicense LICENSE
```
Comment (abenson) [inline srcpkgs/pueue/template:14]: Use a tab here instead of spaces

**Comment by abenson** [comment]
Capitalize the first letter of the commit message (`New package:...`)

**Comment by abenson** [comment]
Bump this to 0.6.1

**Comment by abenson** [comment]
Fix the commit message, too...

**Comment by crater2150** [comment]
Totally missed that, sorry. That's the downside of having an alias for `commit --amend --reuse-message`...

**Pull request merged**